src/expr/src/scalar/func/macros.rs

```diff
@@ -139,7 +139,10 @@ macro_rules! sqlfunc {
         }
     };
 
-    // First, expand the function name into an attribute, if it was omitted
+    // Here follow 3 stages of transformations for the cases where the user didn't provide a fully
+    // specified function signature as expected by the rule above.
+
+    // Stage 0:, expand the function name into an attribute, if it was omitted
```

Comment:
cosmetic nit: `Stage 0:, expand` looks like it should maybe be `Stage 0: Expand`.

Not worth going back through CI for, but next time you're in this file would be nice to clean up.

Reply:
it's ok, I fixed it now so that I don't forget and enabled auto-merge :)